korman/exporter/material.py

```diff
@@ -109,6 +109,8 @@ def _update(self, other):
 class MaterialConverter:
     def __init__(self, exporter):
         self._obj2mat = {}
+        self._bumpMats = {}
+        self._bumpLUT = None
```

Comment:
The same PRP object? We're talking like 1 KB of image data here, so I'm not too concerned with being conservative.